packages/kotti-ui/source/next/kotti-field-select/hooks.ts

```diff
@@ -131,6 +131,44 @@ const useSelectInputSizeFix = <DATA_TYPE extends Values>({
 	})
 }
 
+/**
+ * chrome 66+ shows suggestions for saved autofill data,
+ * through intelligent detection, recognizing "address" fields, for instance,
+ * and suggesting autofills.
+ *
+ * using `autocomplete="off" on `input` of `type="search"` would not work to disable autofill
+ *
+ * HACK to exclude the input field from being auto-fillable
+ * @see {@link https://gist.github.com/niksumeiko/360164708c3b326bd1c8#gistcomment-2032962 }
+ *
+ * could potentially break in even newer versions of chrome.
+ * could be solved by accepting an autocomplete prop
+ * and using tokens for commonly saved autofill fields (e.g. address-line-1)
+ * @see {@link https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-expectation-mantle}
```

Comment:
Love the quote

> The "off" keyword indicates either that the control's input data is particularly sensitive (for example the activation code for a nuclear weapon); or that it is a value that will never be reused (for example a one-time-key for a bank login) and the user will therefore have to explicitly enter the data each time, instead of being able to rely on the UA to prefill the value for them; or that the document provides its own autocomplete mechanism and does not want the user agent to provide autocompletion values.

So Google risks accidentally triggering nukes with ignoring `autocomplete="off"`

Reply:
tagging south korea for ideas about accidental oopsies